### ISSUE TITLE
What does the meaning of count attribute in the response of this link -> `https://developer.allegro.pl/documentation#operation/getCustomerReturns` ?

### ISSUE THREAD

**Issue by mohan-mandal-pattern** [allegro/allegro-api#8702]
I was tasked with implementing pagination for an endpoint in our application. However, unlike other endpoints, the customer returns endpoint does not send a `totalCount` attribute. Because of this, I am a bit confused about the meaning of the `count` attribute in the response for the customer returns endpoint. 
Please help me on this question

**Comment by Lukasz-Zurek** [comment]
Thank you for your message. Yes, you are right, we do not return "totalCount" field in the response on this resource but on the other resources we return it. Please note that this resource is still in Beta version. I understand your needs so I will discuss this case with another department and I will ask about it. If I receive any additional details, I will let you know in this thread.

**Comment by mohan-mandal-pattern** [comment]
@Lukasz-Zurek here the meaning of the `count` attribute is the same as the meaning in the response of the other endpoints?

**Comment by Lukasz-Zurek** [comment]
Yes, it is similar to the `count` in the [GET /sale/offers](https://developer.allegro.pl/documentation#operation/searchOffersUsingGET) resource.

**Comment by mohan-mandal-pattern** [comment]
Cool!
